src/System.Numerics.Tensors/pkg/System.Numerics.Tensors.pkgproj

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+  <PropertyGroup>
+    <!-- we need to be supported on pre-nuget-3 platforms (Dev12, Dev11, etc) -->
+    <MinClientVersion>2.8.6</MinClientVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Numerics.Tensors.csproj" />
+    <ProjectReference Include="..\src\System.Numerics.Tensors.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <HarvestIncludePaths Include="ref/netstandard1.1;lib/netstandard1.1" />
```

Comment:
You shouldn't be harvesting, this is only for bringing in assets from a previous release of a package that you're no longer building.